packages/web-app-files/src/helpers/share/role.ts

```diff
@@ -193,6 +207,41 @@ export const linkRoleUploaderFolder = new LinkShareRole(
   $gettext('uploader'),
   [SharePermissions.create]
 )
+export const spaceViewer = new SpaceShareRole(
+  'spaceViewer',
```

Comment:
I think it's still possible to call it `viewer`, right? The `space` prefix should not be needed.